### ISSUE TITLE
Missing type for RenderHookOptions

### ISSUE THREAD

**Issue by jonkoops** [testing-library/react-hooks-testing-library#534]
- `react-hooks-testing-library` version: 4.0.0
### Relevant code or config:

```js
import { RenderHookOptions } from '@testing-library/react-hooks'
```

### What you did:
Import the `RenderHookOptions` type.

### What happened:

The `RenderHookOptions` type was not found.

### Reproduction:
See code.

### Problem description:
Since the library has been re-written in TypeScript it no longer exports the `RenderHookOptions` as a seperate type. This is causing our CI to fail: https://github.com/Amsterdam/amsterdam-styled-components/pull/1469

### Suggested solution:

Restore the `RenderHookOptions` type so it can be used again.

**Comment by joshuaellis** [comment]
This is solved in `v5.0.0-beta.9` but it also does seem to be cropping up more and I appreciate some people don't want to use a beta library.

**Comment by jonkoops** [comment]
Thanks for the quick response @joshuaellis :+1:
